### DISCUSSION TITLE
Install location for extensions

### DISCUSSION THREAD

**Discussion by Quadrubo** [mlocati/docker-php-extension-installer#866] [Q&A]
Hello,
I'm writing a Dockerfile. In that dockerfile I currently run the command `install-php-extensions` twice. Once in a builder image (for installind dependencies) and another time in the main image where composer, etc are stripped.

Does the command only install the .so file? Is it enough copying the .so file to the new image? What files would I need to copy to omit the second run of the command?

**Comment by mlocati** [comment]
> Is it enough copying the .so file to the new image?

Nope: compiled .so files also depends on system (apt/apk) packages that also need to be installed. And some extension requires manual commands.
